life_expectancy/pipeline.py

```diff
@@ -1,17 +1,80 @@
 """Main with the connection of all needed modules"""
 
 import argparse
-from life_expectancy.load_save_data import load_data, save_data
+from enum import Enum
+from life_expectancy.load_save_data import LoadJsonStrategy, LoadCsvStrategy, save_data
 from life_expectancy.cleaning import clean_data
 
-def main(region = 'pt') ->None:
-    raw_data = load_data()
+class Country(Enum):
+    """List of possible countries"""
+    Austria = 'AT'
+    Belgium = 'BE'
+    Bulgaria = 'BG'
+    Switzerland = 'CH'
+    Cyprus = 'CY'
+    Czech_Republic = 'CZ'
+    Denmark = 'DK'
+    Estonia = 'EE'
+    Greece = 'EL'
+    Spain = 'ES'
+    Finland = 'FI'
+    France = 'FR'
+    Croatia = 'HR'
+    Hungary = 'HU'
+    Iceland = 'IS'
+    Italy = 'IT'
+    Liechtenstein = 'LI'
+    Lithuania = 'LT'
+    Luxembourg = 'LU'
+    Latvia = 'LV'
+    Malta = 'MT'
+    Netherlands = 'NL'
+    Norway = 'NO'
+    Poland = 'PL'
+    Portugal = 'PT'
+    Romania = 'RO'
+    Sweden = 'SE'
+    Slovenia = 'SI'
+    Slovakia = 'SK'
+    Germany = 'DE'
+    Albania = 'AL'
+    Ireland = 'IE'
+    Montenegro = 'ME'
+    North_Macedonia = 'MK'
+    Serbia = 'RS'
+    Armenia = 'AM'
+    Azerbaijan = 'AZ'
+    Georgia = 'GE'
+    Turkey = 'TR'
+    Ukraine = 'UA'
+    Belarus = 'BY'
+    United_Kingdom = 'UK'
+    Moldova = 'MD'
+    Russia = 'RU'
+
+
+def main(region='pt', json_csv='csv') -> None:
+    strategies = {
+        'json': LoadJsonStrategy(),
```

Comment:
`LoadZippedJsonStrategy`

If you receive a Path, you can use Path.suffix to extract your strategy key